### PR TITLE
Update apb-base to support passing mounted secrets to ansible

### PR DESCRIPTION
**Pull request by fabianvf**
Apb base will now add all secrets in /etc/apb-secrets to a temporary yaml file and pass that file into ansible-playbook

https://trello.com/c/oPvl3Rdy
Related:
https://github.com/openshift/ansible-service-broker/pull/345

### CODE REVIEW

**Comment by fabianvf** [comment]
[(13) Ability to read secrets from file/env variables in the broker](https://trello.com/c/oPvl3Rdy/377-13-ability-to-read-secrets-from-file-env-variables-in-the-broker)

**Review by shawn-hurley — APPROVED** [review]
Pending removal of ansible-galaxy

File: apb-base/files/usr/bin/entrypoint.sh [hunk 1]
```diff
@@ -34,20 +34,45 @@ if ! whoami &> /dev/null; then
...
+
+set +x
+
+SECRETS_DIR=/etc/apb-secrets
+mounted_secrets=$(ls $SECRETS_DIR)
+
+extra_args=""
+if [[ ! -e "$mounted_secrets" ]] ; then
+
+    echo '---' > /tmp/secrets
```
Comment (fabianvf) [inline apb-base/files/usr/bin/entrypoint.sh:46]: I wasn't totally sure about writing this to a file, but without some kind of transformation there isn't really a way to get the secrets into ansible, and I thought a file would be better than keeping it on the command line where it's likely to be echoed somewhere

Comment (dymurray) [reply apb-base/files/usr/bin/entrypoint.sh:46]: +1 makes sense

**Review by dymurray — APPROVED** [review]
ACK

**Pull request merged**